### PR TITLE
auto repairs malformed relaystatistics file

### PR DESCRIPTION
**Pull request by x85446**
Two features: Removes old deviceoswd watchdog.   Fixes the bug where deviceID shows up as N/A when a deviceID really exists.

### CODE REVIEW

**Review by whygoyal — APPROVED** [review]

File: wigwag/system/bin/info [hunk 1]
```diff
@@ -370,21 +420,26 @@ _memperf(){
 }
 
 performance(){
-	_placeTitle "Key Process Memory Infomation"
+	_placeTitle "Key Process Memory Infomation (MB)"
 	_placeLine "  Stats in MB and time" "Uptime\t\tPss\tShared\tRss\tVirtual"
-	_memperf "runner" "node /wigwag/devicejs-core-modules/Runner/start.js"
-	_memperf "devicejs-user" "=user"
+	#_memperf "runner" "node /wigwag/devicejs-core-modules/Runner/start.js"
```
Comment (cmonr) [inline wigwag/system/bin/info:425]: Any reason for keeping these around if we can already reference the commit history?

File: wigwag/system/bin/info [hunk 2]
```diff
@@ -17,19 +17,74 @@
...
 LogToTerm=1
 loglevel=info;
-if [[ -e "/wigwag/system/lib/bash/relaystatics.sh" ]]; then
-	source "/wigwag/system/lib/bash/relaystatics.sh"
-else
-	if [[ -e /userdata/edge_gw_config/identity.json ]]; then
-		/wigwag/system/bin/json2sh /userdata/edge_gw_config/identity.json /wigwag/system/lib/bash/relaystatics.sh
+regenerateStats=0;
+
+_check_stat_var_set_error=0;
```
Comment (cmonr) [inline wigwag/system/bin/info:27]: Should this be reset at line 43, or is load_statistics only ever called once?

File: wigwag/system/bin/info [hunk 3]
```diff
@@ -141,35 +196,30 @@ system() {
...
-	if [[ $dwd_active -eq 0 ]]; then
-		WDOGPID="active"
-	elif [[ $dwd_file -ne 0 ]]; then
-		WDOGPID=""
-	elif [[ $dwd_enabled -ne 0 ]]; then
-		WDOGPID="disabled"
-	else
-		WDOGPID=""
-	fi
+	readarray -t IPADDRESSES <<< $IPADDRESS
```
Comment (cmonr) [inline wigwag/system/bin/info:199]: Leaving a comment for future me.

`readarray` doesn't potentially hang in the same way that `read` would if stdin is empty.

**Review by cmonr — COMMENTED** [review]
Some questions, but otherwise LGTM

**Pull request merged**